config/Dockerfiles/Makefile

```diff
@@ -1,15 +1,15 @@
 PARENT_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST)))/../../)
 
-TARGETS := fedora27 centos7 fedora26
+DISTROS := fedora27 centos7 fedora26
```

Comment:
aren't we doing fedora28 yet ?

Reply:
quick question . what version of centos 7.x are we using here ? is it the latest 7.5 ?

Reply:
not yet. That will come after this is in place. There is no container for it yet.

Reply:
For centos, it's whatever is in the container. Off-hand, I don't know specifically, but I think it's at least 7.4

Reply:
@samvarankashyap for reference, you may wish to read the [README](https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/config/README.rst) which details how the DISTRO/Dockerfile structure works.